Jenkinsfile

```diff
@@ -1,55 +1,28 @@
-def formElementPathVersion
-node('hi-speed') {
-    stage('plugin') {
-        dir('plugin') {
-            checkout scm
-            timeout(30) {
-                withMaven(jdk: 'Oracle JDK 8 (latest)',
-                        maven: 'Maven 3.2.1') {
-                    sh 'mvn clean install -B'
-                    formElementPathVersion = sh script: 'cat target/form-element-path/META-INF/MANIFEST.MF | grep Implementation-Version | cut -f 2 -d ":" | xargs', returnStdout: true
-                }
+def mavenName = 'mvn'
+def jdkName = 'jdk7'
+
+node('highram&&docker') {
+    stage 'plugin'
+    dir('plugin') {
+        checkout scm
+        timeout(30) {
+            withMaven(jdk: jdkName, maven: mavenName) {
```

Comment:
This relies on the [Pipeline Maven plugin](https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Maven+Plugin) being present, and it is currently not in ci.jenkins.io.


I can be convinced to install it, as @rsandell has harassed me about it before :smile:  but that would require an INFRA ticket to be thrown onto the backlog.